tests/plugins/reject_odd_funding_amounts.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+"""Simple plugin to test the openchannel_hook.
+
+We just refuse to let them open channels with an odd amount of millisatoshis.
+"""
+
+from lightning import Plugin, Millisatoshi
+
+plugin = Plugin()
+
+
+@plugin.hook('openchannel')
+def on_openchannel(openchannel, plugin):
+    print("{} VARS".format(len(openchannel.keys())))
```

Comment:
You could just include `from pprint import pprint` which recursively prettyprints whatever you pass to it.

Reply:
Nice idea!

Reply:
Doesn't work as well due to undefined ordering, however, and the fact that it prints "{" and "}":

```
b"2019-05-20T22:47:55.079Z plugin-reject_odd_funding_amounts.py {'channel_flags': 1,"
b"2019-05-20T22:47:55.079Z plugin-reject_odd_funding_amounts.py  'channel_reserve_satoshis': '1000000msat',"
b"2019-05-20T22:47:55.079Z plugin-reject_odd_funding_amounts.py  'dust_limit_satoshis': '546000msat',"
b"2019-05-20T22:47:55.079Z plugin-reject_odd_funding_amounts.py  'feerate_per_kw': 7500,"
```